drivers/flash/flash_stm32_ospi.c

```diff
@@ -0,0 +1,1500 @@
+/*
+ * Copyright (c) 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT st_stm32_ospi_nor
+
+#include <errno.h>
+#include <kernel.h>
+#include <toolchain.h>
+#include <arch/common/ffs.h>
+#include <sys/util.h>
+#include <soc.h>
+#include <drivers/pinctrl.h>
+#include <drivers/clock_control/stm32_clock_control.h>
+#include <drivers/clock_control.h>
+#include <drivers/flash.h>
+#include <dt-bindings/flash_controller/ospi.h>
+
+#include "spi_nor.h"
+#include "jesd216.h"
+
+#include <logging/log.h>
```

Comment:
```suggestion
#include <errno.h>
#include <zephyr/kernel.h>
#include <toolchain.h>
#include <zephyr/arch/common/ffs.h>
#include <zephyr/sys/util.h>
#include <soc.h>
#include <zephyr/drivers/pinctrl.h>
#include <zephyr/drivers/clock_control/stm32_clock_control.h>
#include <zephyr/drivers/clock_control.h>
#include <zephyr/drivers/flash.h>
#include <zephyr/dt-bindings/flash_controller/ospi.h>

#include "spi_nor.h"
#include "jesd216.h"

#include <zephyr/logging/log.h>
```
Cf https://github.com/zephyrproject-rtos/zephyr/pull/45410

Reply:
changed : include path